tailcfg/derpmap.go

```diff
@@ -7,6 +7,11 @@ import "sort"
 
 // DERPMap describes the set of DERP packet relay servers that are available.
 type DERPMap struct {
+	// Hints, if non-nil, is a change in hints.
+	//
+	// The rest of the DEPRMap fields, if zero, means unchanged.
```

Comment:
maybe clarify that OmitDefaultRegions is only applicable when Regions is non-nil?